llama_index/storage/storage_context.py

```diff
@@ -28,6 +28,14 @@
 
 DEFAULT_PERSIST_DIR = "./storage"
 
+# default storage context. Inherited from when calling `from_defaults`.
+default_storage_context: Optional["StorageContext"] = None
```

Comment:
is this used anywhere?

Reply:
Not at the moment. Not sure if we want this (or even the inheritance default)